cluster/manifests/deployment-service/status-service-ingress.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     component: status-service
 spec:
   hosts:
+    - depl-status-{{ .Cluster.Alias }}.{{ .Values.hosted_zone }}
```

Comment:
Do you think it's worth it to spend 2 more characters on `deploy` instead of `depl` for the sake of naming clarity?

Reply:
Adding 2 more characters makes it 1 character too long for the longest current cluster name :facepalm:

Reply:
🫠